azure-pipelines.yml

```diff
@@ -121,9 +122,6 @@ stages:
               - hdf5-devel
 
           envs:
-            - linux32: py38-32bit
```

Comment:
I cracked and killed it.

Reply:
I mean it makes my life easier if I kill it on the other repos too.

Reply:
Only one where it's potentially useful is sunkit-image and pyflact as there is a lot of C

Reply:
I might keep it around there.